qa/L0_batcher/batcher_test.py

```diff
@@ -264,11 +311,13 @@ def test_multi_batch_not_preferred(self):
                 threads.append(threading.Thread(target=self.check_response,
                                                 args=(trial, 1,
                                                       (_max_queue_delay_ms * 1.5, _max_queue_delay_ms)),
-                                                kwargs={'shm_region_names': ['ip00', 'ip01', 'op00', 'op01']}))
+                                                kwargs={'shm_region_names': ['ip00', 'ip01', 'op00', 'op01'],
+                                                'precreated_shm_regions': precreated_shm_regions[:2]}))
```

Comment:
Each time _create_advance is called the clients needs to get the SharedMemoryControlContext from the server. I did this to avoid the same for each request. I preregister all regions and run inference on them all at once.

Reply:
Done